api/v1alpha1/wasm_types.go

```diff
@@ -49,9 +49,9 @@ type Wasm struct {
 	// +kubebuilder:default=false
 	FailOpen *bool `json:"failOpen,omitempty"`
 
-	// Priority defines the location of the Wasm extension in the HTTP filter chain.
```

Comment:
I think that this should not change in this PR. This priority is related to location in the filter chain, not backend failover.